Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -70,6 +70,26 @@ public class MixedRealityInputManager : MixedRealityEventManager, IMixedRealityI
 
 #endif // UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
 
+        /// <summary>
+        /// Current Speech Input Source.
+        /// </summary>
+        public SpeechInputSource SpeechInputSource { get; private set; }
+
+        /// <summary>
+        /// Current Dictation Input Source.
+        /// </summary>
+        public DictationInputSource DictationInputSource { get; private set; }
+
+        /// <summary>
+        /// Current Touch Screen Input Source.
+        /// </summary>
+        public TouchscreenInputSource TouchscreenInputSource { get; private set; }
```

Comment:
A question was raised by a customer this week wrt the automatic implementation of inputs like touch screen. They would like the ability to completely shut off input sources that they never intend to use.

Seems like we could leverage our configuration profiles to add check boxes (defaulted to checked) specifying each input type (mouse, touch screen, controller, etc.) and allow the dev to disable specific source types.

Reply:
Great idea, I'll be sure to add that